code/modules/mob/living/say.dm

```diff
@@ -144,12 +144,25 @@ proc/get_radio_key_from_channel(var/channel)
 	else
 		speaking = get_default_language()
 
+	// Check if the speech requires dexterity, and if so, stop if can't use hands.
+	if(speaking && (speaking.flags & NEEDS_DEX))

```

Comment:
Also pretty snowflakey.

This should all be handled on the grey's language, now snowflaked into all of mob say code.